src/ontology/mondo-ingest.Makefile

```diff
@@ -199,6 +199,14 @@ $(REPORTDIR)/%_mapping_status.tsv $(REPORTDIR)/%_unmapped_terms.tsv: $(REPORTDIR
 	--outpath-simple $(REPORTDIR)/$*_unmapped_terms.tsv \
 	--outpath-full $(REPORTDIR)/$*_mapping_status.tsv
 
+components/%-unmapped.owl: components/%.owl reports/%_unmapped_terms.tsv
+	cut -f 1 reports/$*_unmapped_terms.tsv | tail -n +2 > reports/$*_unmapped_terms.txt
```

Comment:
Never overwrite another file in a make goal - either add this to the reports/%_unmapped_terms.tsv goal, or create an entirely new file here that you feed into the filter.

Reply:
Hmm, makes sense, but technically I'm actually not overwriting anything (you might have misread `.tsv` as `.txt` in the first part of the goal). As in your latter suggestion, I'm creating a new file and then feeding it to the filter.

The whole goal is:
```make
	cut -f 1 reports/$*_unmapped_terms.tsv | tail -n +2 > reports/$*_unmapped_terms.txt
	$(ROBOT) filter -i components/$*.owl -T reports/$*_unmapped_terms.tsv -o $@
	rm reports/$*_unmapped_terms.txt
```

We didn't previously have `reports/*_unmapped_terms.txt`. We had `reports/*_mapping_status.tsv` (`subject_id`, `subject_label`, `is_mapped`, `is_excluded`, `is_deprecated`) and `reports/*_unmapped_terms.tsv` (`subject_id`, `subject_label`). So, the only goal which uses this new `reports/*_unmapped_terms.txt` is `components/%-unmapped.owl`; So I just create it on the fly and then delete it afterwards.

Wondering if you would like me to:
(a) keep doing this
(b) Put it in `tmp/` instead since it's just deleted immediately
(c) If we want to commit `reports/*_unmapped_terms.txt` as well, I can add it to the same goal that makes those other 2 mapping artefacts.